### ISSUE TITLE
Automate undoing of latest deployment

### ISSUE THREAD

**Issue by conbrad** [bcgov/wps#1777]
**Describe the task**
We manually fix deployments and it is inefficient and time consuming. Automate this process with a one line script.

**Acceptance Criteria**
- [ ] Any dev should be able to re-deploy previous deployment with a one line script

**Additional context**
- Add any other context about the task here.
- Or here

**Comment by Sybrand** [comment]
I don't think we need a script for this? Given it's a one line command?  Let's discuss.

**Comment by conbrad** [comment]
Yeah I think we can close this when https://github.com/bcgov/wps/pull/1778 is merged
